tests/unit_tests/test_tethys_apps/test_base/test_permissions.py

```diff
@@ -105,33 +105,33 @@ def test_has_permission_no(self, mock_app):
         self.assertFalse(result)
 
 
-class TestAsyncPermissionGroup(unittest.IsolatedAsyncioTestCase):
-    def setUp(self):
-        self.user = UserFactory()
-        self.request_factory = RequestFactory()
-        self.name = "test_name"
-        self.permissions = ["foo", "bar"]
-        self.check_string = '<Group name="{0}">'.format(self.name)
-
-    def tearDown(self):
-        pass
-
-    @mock.patch("tethys_apps.utilities.get_active_app")
-    async def test_scoped_user_has_permission(self, mock_app):
-        self.user.has_perm = mock.MagicMock(return_value=True)
-        scope = {"user": self.user, "path": "some/url/path"}
-        mock_app.return_value = mock.MagicMock(package="test_package")
-        result = await tethys_permission.scoped_user_has_permission(
-            scope=scope, perm="test_perm"
-        )
-        self.assertTrue(result)
-
-    @mock.patch("tethys_apps.utilities.get_active_app")
-    async def test_scoped_user_has_permission_no(self, mock_app):
-        self.user.has_perm = mock.MagicMock(return_value=False)
-        scope = {"user": self.user, "path": "some/url/path"}
-        mock_app.return_value = mock.MagicMock(package="test_package")
-        result = await tethys_permission.scoped_user_has_permission(
-            scope=scope, perm="test_perm"
-        )
-        self.assertFalse(result)
+# class TestAsyncPermissionGroup(unittest.IsolatedAsyncioTestCase):
```

Comment:
Why are these commented out? Do we need to fix them? Remove them?

Reply:
These tests are obsolete and should be removed. I initially commented them out because I thought the would be a helpful reference for running async tests against the mixins module where the tested functionality was moved to.

Reply:
If you are done with these commented out tests, please remove.